packages/projects-docs/pages/learn/teams/_meta.json

```diff
@@ -1,5 +1,6 @@
 {
   "team-overview": "Working with Teams",
   "permissions": "User Permissions",
-  "preferences": "Preferences"
+  "preferences": "Preferences",
+  "workspace": "Workspace"
 }
```

Comment:
We are probably missing an entry here for the "Plans & Billing" menu entry, which is currently displaying as `plan-billing`

Reply:
thanks for spotting this